MMA8452.cpp

```diff
@@ -517,3 +517,25 @@ int8_t MMA8452::convertTo2sComplement(int8_t value)
 	value = (0xFF && ~value) + 1;
 	return value;
 }
+
+void MMA8452::getPitchRoll(float *pitch, float *roll)
```

Comment:
Could this also support yaw?

Reply:
I am not aware of a way to calculate yaw without a magnetometer or gyro.